app/models/OutputConfiguration.ts

```diff
@@ -0,0 +1,10 @@
+export type OutputConfiguration = {
+    type: string;
+    name: string;
+    storageRelativePath: string;
+}
+export class OutputConfiguration {
+    constructor(outputConfiguration: OutputConfiguration) {
+        this.name = outputConfiguration.name;
+    }
+}
```

Comment:
Remove this file and instead add the name field in the `IOutputConfiguration` interface. It's in `app/outputs/IOutputConfiguration.ts`. There's no need for a constructor, you can consider the name field initialized.